File: Src/Client/src/app/shared/models/activity-group-user.model.ts
export class ActivityGroupUserModel {
  id: string;
  activityGroupId: string;
  userId: number;
  role: TsRole;
  fullName: string;
  email: string;
  constructor(init?: Partial<ActivityGroupUserModel>) {
    Object.assign(this, init);
  }
}

export enum TsRole {
  MANAGER = 1,
  QA = 2,
}

File: Src/Server/Kloon.EmployeePerformance.Models/TimeSheet/ActivityGroupUserModel.cs
using Kloon.EmployeePerformance.Models.Common;
using System;

namespace Kloon.EmployeePerformance.Models.TimeSheet
{
    public class ActivityGroupUserModel
    {
        public Guid Id { get; set; }
        public Guid ActivityGroupId { get; set; }
        public int UserId { get; set; }
        public TSRole Role { get; set; }
        public string FullName { get; set; }
        public string Email { get; set; }
    }
}
